include/class_loader/meta_object.hpp

```diff
@@ -69,7 +69,7 @@ class CLASS_LOADER_PUBLIC AbstractMetaObjectBase
    * TEMPLATE SUBCLASSES, OTHERWISE THEY WILL PULL IN A REDUNDANT METAOBJECT
    * DESTRUCTOR OUTSIDE OF libclass_loader WITHIN THE PLUGIN LIBRARY! T
    */
-  ~AbstractMetaObjectBase();
+  virtual ~AbstractMetaObjectBase();
```

Comment:
@y-okumura-isp meta: the comment is rather imprecise. Lacking `virtual` indeed breaks polymorphic destruction.